governance.md

```diff
@@ -0,0 +1,57 @@
+# Governance
+
+## Issue Reporting
+
+Issues should be reported in accordance with the [contribution guidelines](contributing.md).
+
+## Issue Review
+
+Reported issues are reviewed as quickly as possible.  After review, a response will be made indicating potential or planned next steps.  Such next steps may include asking for additional information or indicating a bug was discovered.
+
+If an update is required to a dependency project, then the `fqm-execution` team will indicate which dependency update is required and may assist that team in the resolution. 
+
+## Releases
+
+An agile methodology using 2-week sprints is used for `fqm-execution`.  If a change is required to `fqm-execution` to resolve an issue, then the issue will be assigned as a task in sprint planning for development work.  This will lead to inclusion in a future release of `fqm-execution`.
+
+Issue resolutions are made in pull requests to `fqm-execution`.  Observers are invited to review pull requests for comment or testing.
+
+New releases are generally scheduled for the Wednesday at the end of a sprint.  Releases, such as urgent hotfix releases, may be made out-of-cycle at the `fqm-execution` team's discretion.  In all cases, a release will include information on issues resolved.
+
+## Closing an Issue
+
+Issues are closed upon resolution or inactivity.  This includes the following:
+- An issue may require only information.  In this case, the issue will be closed after the information is provided.
+- An issue may require an update to `fqm-execution`.  In this case, the issue will be closed after a pull request has been accepted resolving the issue.  The next release of `fqm-execution` will include the update.
+- An issue may be filed with no follow up activity from the submitter (e.g. more information is required, but not provided).  In this case, the issue will be closed and a new issue will need to be filed to revisit if necessary.
```

Comment:
We could use another bullet here for cases where the issue is actually a problem in `cql-execution` or the `cql-to-elm` translator. Often I think we could create tickets in those repositories. If it is a `cql-execution` issue we will create the ticket in the project and reference it. When the fix is released in `cql-execution` we will update our package's reference and update our version, then close the issue here.

Reply:
Perfect!